alcs-frontend/src/app/features/application/review/decision-meeting-dialog/decision-meeting-dialog.component.html

```diff
@@ -13,8 +13,8 @@ <h2 class="card-title">{{ model.uuid ? 'Update' : 'Create' }} Decision Meeting</
   </mat-dialog-content>
   <mat-dialog-actions align="end">
     <div>
-      <button mat-button mat-dialog-close>Close</button>
-      <button mat-button type="submit" [disabled]="!meetingForm.form.valid">Save</button>
+      <button style="margin-right: 24px !important;" mat-stroked-button color="primary" mat-dialog-close>Close</button>
```

Comment:
Why not a separate class in typography for button styling?

Reply:
Good idea, added a .button-container in styles.scss and used that